homeassistant/components/threshold/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for Threshold integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_ENTITY_ID, CONF_NAME
+from homeassistant.helpers import selector
+from homeassistant.helpers.helper_config_entry_flow import (
+    HelperConfigFlowHandler,
+    HelperFlowError,
+    HelperFlowStep,
+)
+
+from .const import CONF_HYSTERESIS, CONF_LOWER, CONF_UPPER, DEFAULT_HYSTERESIS, DOMAIN
+
+
+def _validate_mode(data: Any) -> Any:
+    """Validate the threshold mode."""
+    if data[CONF_LOWER] is None and data[CONF_UPPER] is None:
+        raise HelperFlowError("need_lower_upper")
+    return data
+
+
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HYSTERESIS, default=DEFAULT_HYSTERESIS): selector.selector(
```

Comment:
The default isn't showing in the UI. Is that a bug?

![CleanShot 2022-03-18 at 19 59 28@2x](https://user-images.githubusercontent.com/195327/159067642-005b537b-6645-402e-a14e-92a42f83e32e.png)

Reply:
Yeah latest dev, but its not relevant for PR I guess